test/base/test0196.d

```diff
@@ -39,9 +46,9 @@ void main() {
 	enum NodeCount = 10000000;
 
 	foreach (loop; 0 .. 20) {
-		auto ll = new Link(null);
+		auto ll = allocLink(null);
```

Comment:
Note, I realized on linux, when we are not using the SDC gc as the main gc, this actually allocates using the old gc! and so it's not testing anything.